pkg/interface/chat/src/js/components/new.js

```diff
@@ -124,6 +124,10 @@ export class NewScreen extends Component {
       }
     });
 
+    if(state.ships.length === 1 && state.security === 'village' && !state.createGroup) {
+      props.history.push(`/~chat/new/dm/${aud[0]}`);
+    }
+
     if (!isValid) {
```

Comment:
While we're making changes in new.js, let's remove the "unmanaged invite-only chat" case.

Reply:
Can you elaborate?

Reply:
Essentially we're simplifying over time: 
- Unmanaged chats will be synonymous with direct messages, which are invite-only and created through the DM flow
- All other chats people are in will be within a group

**Right now** since we don't have open groups you can just paste a shortcode into, people use "lobby chats" (public unmanaged chats). By removing the invite-only unmanaged chat flow from new.js, we start to simplify these distinctions until groups are updated and we can remove unmanaged chats from the new.js screen entirely.

Reply:
It would mean, in this case, removing the invite only toggle and assuming it's always a public chat when we make a new unmanaged chat from the "New Chat" screen (managed chats stay invite only of course).